boards/arm/lpcxpresso55s69/lpcxpresso55s69.dtsi

```diff
@@ -90,6 +90,45 @@
 	current-speed = <115200>;
 };
 
+&flash0 {
+	/*
+	 * For more information, see:
+	 * http://docs.zephyrproject.org/latest/guides/dts/index.html#flash-partitions
+	 */
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x00000000 0x00010000>;
+			read-only;
+		};
+		/*
+		 * The flash starting at 0x10010000 and ending at
```

Comment:
@agansari is this reserved for TF-M, or for some other image, or library?

Reply:
@ioannisg not sure who uses it, tfm or mcuboot, when creating this partitioning scheme i followed the template of other devices.
Most devices (aside from Nordic) have a range reserved for applications and are not part of a partition, I see Nordic use the storage partition for applications.

Reply:
I suspect it isn't used for anything general on Zephyr. The intent was to be an area for something like a filesystem, or the app to store some kind of state.

Reply:
Will merge this un-partitioned space into storage partition.